src/main/java/net/gnomeffinway/depenizen/events/PVPArenaEvents.java

```diff
@@ -41,7 +43,7 @@ public void onArenaStarts(PAStartEvent event) {
         context.put("fighters", new dList(fighters));
 
         OldEventManager.doEvents(Arrays.asList
-                        ("arena starts"),
+                        ("pvparena starts"),
                 null, context);
```

Comment:
I wanted to make the pvparena event unique so it wasn't confused with other events.